autoPyTorch/utils/backend.py

```diff
@@ -329,6 +333,25 @@ def get_runs_directory(self) -> str:
     def get_numrun_directory(self, seed: int, num_run: int, budget: float) -> str:
         return os.path.join(self.internals_directory, 'runs', '%d_%d_%s' % (seed, num_run, budget))
 
+    def get_next_num_run(self, peek: bool = False) -> int:
+
```

Comment:
could you add a little docstring to explain what peek does? And along with this maybe also for the function